### PR TITLE
add metadata to docs

### PR DESCRIPTION
**Pull request by novikov82**

### CODE REVIEW

File: docs/get-started-angular.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+---
+title: Knockout Form Library | Getting Started Guide
+description: A step-by-step tutorial on how to add the SurveyJS Form Library to a Knockout application.
+---
```
Comment (RomanTsukanov) [inline docs/get-started-angular.md:4]:
```suggestion
```

File: docs/get-started-angular.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
...
+---
+title: Angular Form Library | Getting Started Guide
+
+
```
Comment (RomanTsukanov) [inline docs/get-started-angular.md:8]:
```suggestion
title: Angular Form Library | Getting Started Guide
```

File: docs/icons.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
+---
+title: UI icons | SurveyJS Form Libraries
+description: View the list of all available built-in icons used in SurveyJS libraries, and the code showing how to replace a built-in icon with a custom one.
```
Comment (RomanTsukanov) [inline docs/icons.md:3]:
```suggestion
description: View the list of all available built-in icons used in SurveyJS libraries and the code showing how to replace a built-in icon with a custom one.
```

File: docs/localization.md [hunk 4]
```diff
@@ -1,3 +1,7 @@
+---
+title: Survey Localization | SurveyJS Form Libraries
+description: To adapt a survey to the local language of your respondents, you can localize its UI elements and survey contents. Leverage our community-sourced support for 50+ languages.
```
Comment (RomanTsukanov) [inline docs/localization.md:3]:
```suggestion
description: To adapt a survey to the local language of your respondents, localize survey UI elements and contents. Leverage our community-sourced support for 50+ languages.
```

File: docs/design-survey-create-a-multi-page-survey.md [hunk 5]
```diff
@@ -1,3 +1,7 @@
+---
+title: Create a Multi-Page Survey | SurveyJS Form Libraries
+description: SurveyJS Form Library lets you add multiple pages to your survey. Learn how to control page visibility and navigation, and configure the introductory and thank you pages.
```
[the remark this answers is not in the document]
Comment (ElenaGorbatkova) [reply docs/design-survey-create-a-multi-page-survey.md:3]: comma is required here. i just double-checked with an editor

**Review by RomanTsukanov — APPROVED** [review]

**Pull request merged**